plexpy/webserve.py

```diff
@@ -1183,7 +1190,7 @@ def user_watch_time_stats(self, user=None, user_id=None, **kwargs):
 
         if user_id or user:
             user_data = users.Users()
-            result = user_data.get_watch_time_stats(user_id=user_id)
+            result = user_data.get_watch_time_stats(user_id=user_id, time_queries=[1, 7, 30, 0])
```

Comment:
Can leave out the `time_queries` argument since the default is added to `get_watch_time_stats`.